src/third-party/gutenberg.stories.mdx

```diff
@@ -0,0 +1,568 @@
+import { Story, Preview, Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Third-Party/Gutenberg" />
+
+# WordPress Gutenberg Blocks
+
+The Wordpress Gutenberg editor is a block-based page builder. The editor comes with
+a variety of core blocks. Of those, a subset automatically generate markup with one
+or more `wp-block-`-prefixed classes. These are the blocks documented below.
+
+Each block has controls that can modify the generated output. Where relevant, the
+additional markup and/or classes that may be generated are also documented.
+
+### Editor Controls
+
+Block editor controls vary, but many have Styles and Color settings. The effects
+of other controls are described in specific block stories.
+
+#### Styles
+
+These controls offer 2 or more options for particular stylistic variations, and
+the ability to set one as a default for new blocks of the same type. For example,
+images can have a circular mask or not. These options add an `is-style-`-type class
+to the block.
+
+#### Color settings
+
+These controls enable setting of the block's `background-color` and `color`.
+
+The base class for a block with a main (background) color is `has-background` and
+selecting a text color adds a `has-text-color` class. If one of a set of predefined
+colors is chosen, additional classes are added, such as `has-subtle-background-background-color`
+for `background-color` or `has-secondary-color` for text.
+
+If user-defined colors are chosen, instead, inline styles will set `color` and/or
+`background-color`.
+
+## Audio
+
+The audio block wraps an `audio` element and an optional `figcaption` element in
+a `figure` with class `wp-block-audio`
+
+<Preview>
+  <Story name="core/audio" height="150px">{`
+    <figure class="wp-block-audio type">
+      <audio controls="" src=""></audio>
+      <figcaption>My audio clip</figcaption>
+    </figure>
+  `}</Story>
+</Preview>
+
+## Button
+
+The button block creates one or more `div`s with class `wp-block-button`, each containing
+an `a` element with a class of `wp-block-button__link`.
```

Comment:
You got it. In this case, `button` is meant to be a link, either to directly get a media asset or navigate to its attachment page. It could probably use a better name.

Reply:
The intent here is to add call-to-action style links to a post. The containers exist for layout, the links to link to that content. I think in this case the naming of this pattern is probably more confusing to web developers than to end users.

Reply:
I can update the description, if that seems worthwhile.